boms/cloud-oss-bom/RELEASING.md

```diff
@@ -14,10 +14,15 @@ tool if you not previously done so.
 
 ## Steps
 
-Run `prepare_release.sh` with `bom` argument in `cloud-opensource-java` directory:
+All on your corp desktop: 
+
+1. Run gcert if you have not done so in the last twelve hours or so.
```

Comment:
```suggestion
1. Run gcert.
```

Shorter is clear.

Reply:
Yes, but even better is not doing it at all, which is often the case. :-)